### ISSUE TITLE
Self Hosting doesn't allow account verification

### ISSUE THREAD

**Issue by BrokenSword808** [HexCodeFFF/captionbot#265]
### Describe the issue

"TypeError: Command raised an exception: TypeError: cannot pickle 'BufferedReader' instances" whenever I try to download anything after putting account credentials into the "nano /mediaforge/src/processing/other.py" part of the debug menu. 

### Expected Behavior

No way of inputting the verification code for twitter/instagram/other sites when requested

### Logs

Not sure what to put here

### Config

not sure about this either

### Hosting Provider

My machine

### Hosting Method

Docker, stock everything

### Hosting Resources

8tb hdd with 64 gigs ddr5, i9-14900k

### Anything Else

Sorry for the lack of info, I'm not a coder and don't know anything about coding either.

**Comment by reticivis-net** [comment]
Yeah putting in credentials isn't a priority feature, and the error your experiencing is a bug with the error handling so I can't really see the true problem. Can you send the code?
